tests/cli/test_integration_datasets.py

```diff
@@ -269,8 +269,26 @@ def test_dataset_reimport_removed_dataset(runner, project, sleep_after):
 
 
 @pytest.mark.integration
```

Comment:
With #902, there is a new decorator with which you can specify integration tests for a rerun

```
@flaky(...)
...
```

This should make IT more robust in case of failure when running this in parallel.